app/posts/publish-teacher-training-courses/2021-06-03-degree-entry-requirements.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Degree entry requirements
+description: Asking providers what their degree entry requirements are
+date: 2021-06-03
+---
+
+{% from "screenshots/macro.njk" import appScreenshots with context %}
+
+Some providers require a minimum degree classification, such as a 2:2, on top of the statutory requirement for a degree. Not meeting this can be a reason why candidates are rejected. Research has also shown that where candidates are aware of this before applying, it can take them a long time to sift through courses in order to find those that will accept their degree classification.
+
+Currently, the requirement for a degree, any minimum classification, and any subject requirements, are all described by providers within a single text box labelled "Qualifications needed".
+
+In order to enable candidates to more easily find courses which will accept their degree classification, we are changing this text box to a set of structured questions and answers.
+
+These questions have to be answered for each course separately, as analysis research showed that whilst most providers have the same degree requirements across all their courses, some have different degree entry requirements depending on the course type.
```

Comment:
Could add : this was also supported by training providers during user research, who gave examples of degree requirements varying between courses, for example subjects without SKE courses.